lib/msal-core/src/PopTokenCommon.ts

```diff
@@ -0,0 +1,61 @@
+export enum HashAlgorithm {
+    sha256 = 'SHA-256'
+};
+
+export enum KeyGenAlgorithm {
+    rsassa_pkcs1_v15 = "RSASSA-PKCS1-v1_5"
+}
+
+export enum KeyUsages {
+    sign = "sign",
+    verify = "verify"
+}
+
+export enum KeyFormat {
+    jwk = "jwk"
+}
+
+export function getStringFromArrayBuffer(data: ArrayBuffer): string {
+    return String.fromCharCode.apply(null, new Uint8Array(data));
+}
+
+export function getArrayBufferFromString(dataString: string): ArrayBuffer {
+    const data = new ArrayBuffer(dataString.length);
+    const dataView = new Uint8Array(data);
```

Comment:
```js
const dataView = dataString.split('').reduce((memo, val, i) => {
    return [...memo, [i]: dataString.charCodeAt(i)];
}, new Uint8Array(data));
```

Reply:
maybe this is more confusing haha

Reply:
Yeah, for loops with mutations isn't my favorite either. Was copy/pasta from the POC, might revisit later.